spacialgaze-plugins/misc.js

```diff
@@ -433,4 +433,26 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+
+	'!ai': true,
+	askai: 'ai',
+	'8ball': 'ai',
+	ai: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		const http = require('http');
```

Comment:
In this case, it should go on top, we dont want to require something everytime a command runs.